vscode/src/paths.ts

```diff
@@ -90,3 +94,96 @@ export function fsPaths(): ExtensionFsPaths {
   }
   return _fsPaths;
 }
+
+const DEFAULT_IGNORES = [".git", ".vscode", "target", "node_modules"];
+const IGNORE_FILE_IN_PRIORITY_ORDER = [".konveyorignore", ".gitignore"];
+
+let _ignoreByFunction: undefined | ((path: string) => boolean);
```

Comment:
Again, for my edification. Is the order of the typing here significant?

Reply:
Not really.  The variables starts off as an undefined and then will become a function once the first call is made.  So the order here is just the order I was writing the code.